modules/ROOT/pages/api-led-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+= Build an API Tutorial: Prerequisites
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:experimental:
+
+Before you begin your API-led journey, verify that you have the basic tools and access needed.
+
+* Username and password for your Anypoint Platform organization. If you don't have one yet, create a Trial organization on Anypoint Platform.
```

Comment:
```suggestion
* Username and password for your Anypoint Platform organization. 
If you don't have one yet, create a Trial organization on Anypoint Platform.
```
...but the "one" in the second sentence bothers me, because it seems it could apply to username, password, organization, or onganization on AP.... What do you think?

Reply:
changed to Anypoint Platform.